packages/samples/mvp-dapp/kuai.config.js

```diff
@@ -7,9 +7,18 @@
 require('dotenv').config()
 
```

Comment:
What if using

```typescript
// fallback to REDISUSER due to https://github.com/ckb-js/kuai/pull/423#issuecomment-1668983983
const REDIS_USER = process.env.REDIS_USER ?? process.env.REDUSUSER
const REDIS_PASSWORD = process.env.REDIS_PASSWORD ?? process.env.REDISPASSWORD
const REDIS_OPT = process.env.REDIS_OPT ?? process.env.REDISOPT
const REDIS_HOST = process.env.REDIS_HOST ?? process.env.REDISHOST
const REDIS_PORT = process.env.REDIS_PORT ?? process.env.REDISPORT
```

`REDIS_*` are kept because they are defined in the `env.example`(https://github.com/ckb-js/kuai/blob/develop/packages/samples/mvp-dapp/.env.example) and work locally. `REDIS*`s only work for railway

Reply:
`REDIS_OPT` includes `username` and `password`. And `railway` does not include the `REDISOPT ` variable.
And we are not defining `REDIS_USER` and `REDIS_PASSWORD`.

Do you mean to write them together and add an annotation?

Reply:
> Do you mean to write them together and add an annotation?

Almost yes, I would like to keep the names of variables consistent so it's clear that they are paired.

Otherwise, I would like to figure out why `port` is `REDIS_PORT ?? REDISPORT` but `username` is `REDISUSER` but not `REDIS_USER`, and why `redis option` is `REDIS_OPT` but not `REDISOPT`

Reply:
Another issue is that

```
REDIS_OPT = {"username":"${{Redis.REDISUSER}}","password":"${{Redis.REDISPASSWORD}}"}
```
is defined in railway configuration, but they are overridden by

```
  redisOpt = {
    username: process.env.REDISUSER,
    password: process.env.REDISPASSWORD,
  }
  redisOpt = {
    ...JSON.parse(process.env.REDIS_OPT),
    ...redisOpt,
  }
```

Reply:
If they define `REDISUSER` and `REDISPASSWORD`. It means they want to use `REDISUSER` and `REDISPASSWORD`.

Reply:
> If they define `REDISUSER` and `REDISPASSWORD`. It means they want to use `REDISUSER` and `REDISPASSWORD`.

If it's a reply to https://github.com/ckb-js/kuai/pull/423#discussion_r1286760953

> Another issue is that
> 
> ```
> REDIS_OPT = {"username":"${{Redis.REDISUSER}}","password":"${{Redis.REDISPASSWORD}}"}
> ```
> is defined in railway configuration, but it is overridden by
> 
> ```
>   redisOpt = {
>     username: process.env.REDISUSER,
>     password: process.env.REDISPASSWORD,
>   }
>   redisOpt = {
>     ...JSON.parse(process.env.REDIS_OPT),
>     ...redisOpt,
>   }
> ```

My point is that `REDIS_OPT` can be left empty by default in railway because `username` and `password` of `REDIS_OPT` is overridden by railway's default `REDISUSER` and `REDISPASSWORD`